SpaDES4Dummies.Rmd

```diff
@@ -209,12 +230,16 @@ abundancePlot <- function(sim) {
 }
 ```
 
+"It lookd great, but... How does it know that it should plot them **after** the abundance generator?" you ask, puzzled. Good question sir/madam, 5 points for Griffindor!
+
+####**3.Scheduling events**{#anchor3}
 
-####Scheduling events
+Events *within a module* will occur discretely in a given order, specified using the `scheduleEvent` function. These, such as `abundanceInit`, `abundancePlot` and `abundanceSim` are also sub-functions inside the `doEvent.speciesAbundance` function.  Events *between or among modules* will obey either the internal module's scheduling (if the time is set differently for each event, it will jump from module to module, obeying the specified time order), or the order stated when loading the modules (if the time is the same for all events across modules, it will run all events from the first loaded module, then the same for the second module, and so forth).
```

Comment:
I think there's something confusing me here... please remind me to discuss it with you, when you go over the revisions :)